zproject/urls.py

```diff
@@ -658,9 +662,9 @@
 # such links with images previews. Now thumbor can be used for serving such
 # images.
 urls += [
-    re_path(r'^external_content/(?P<digest>[\S]+)/(?P<received_url>[\S]+)$',
-            zerver.views.camo.handle_camo_url,
-            name='zerver.views.camo.handle_camo_url'),
+    path('external_content/<digest>/<received_url>',
```

Comment:
Hmm, is this changing the handling of `/` here?

Reply:
`digest` and `received_url` are both hex-encoded (`zerver.lib.camo.generate_camo_url`).

Reply:
OK, great.